packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -468,7 +468,6 @@ export class DropdownBase extends BaseComponent<IDropdownInternalProps, IDropdow
         <FocusZone
           ref={this._focusZone}
           direction={FocusZoneDirection.vertical}
-          defaultActiveElement={selectedIndices[0] !== undefined ? `#${id}-list${selectedIndices[0]}` : undefined}
```

Comment:
Subtle but this caused a regression in one of the screener scenarios. You probably need to still do this in certain cases to handle the initial render is my guess.

Reply:
So the reason this breaks things, I think, is because of how callout positions. This will set the active element when it mounts, but that's not the right time to have it scroll appropriately, then when the onpositioned callback is called it refocuses the same element which won't trigger a scroll since it's already focused.